src/Worker.Extensions.DurableTask/FunctionsDurableTaskClient.cs

```diff
@@ -85,9 +85,10 @@ public override AsyncPageable<OrchestrationMetadata> GetAllInstancesAsync(Orches
     }
 
     public override Task TerminateInstanceAsync(
-        string instanceId, object? output = null, CancellationToken cancellation = default)
+        string instanceId, TerminateInstanceOptions? options = null, CancellationToken cancellation = default)
     {
-        return this.inner.TerminateInstanceAsync(instanceId, output, cancellation);
+        object? output = options?.Output;
+        return this.inner.TerminateInstanceAsync(instanceId, output: output, cancellation);
```

Comment:
We should be passing the options directly to the inner call:

```suggestion
        return this.inner.TerminateInstanceAsync(instanceId, options, cancellation);
```